entries/m/miamioh.edu.json

```
@@ -5,14 +5,17 @@
       "sms",
       "call",
       "custom-software",
-      "custom-hardware"
+      "u2f"
     ],
-    "documentation": "https://miamioh.edu/it-services/initiatives-and-projects/info-security/duo/",
+    "custom-software": [
+      "Duo"
+    ],
```

```suggestion
    ],
    "custom-hardware": [
    	"Feitian OTP c100 Token"
    ],
```

[Source](https://miamioh.teamdynamix.com/TDClient/1813/Portal/KB/ArticleDet?ID=60340)